lib/wasmer-types/src/types.rs

```diff
@@ -229,9 +229,9 @@ impl ExternType {
 #[cfg_attr(feature = "enable-serde", derive(Serialize, Deserialize))]
 pub struct FunctionType {
     /// The parameters of the function
-    params: Vec<Type>,
+    params: Box<[Type]>,
```

Comment:
Does this affect serde somehow?

Reply:
I don't believe it should, but that's a good thing to check -- thanks!

Reply:
Alright, checked the serde source code: they're both serialized as sequences, so there shouldn't be any effect on the serialization in any format